src/backend/efiling-api/README.md

```diff
@@ -1,5 +1,24 @@
 # efiling-api
 
+## Maven Profiles
+
+### Openshift
+
+Current image build in openshift runs with `openshift` profile by default.
+This profiles mostly enables splunk logging.
+
+```bash
+mvn install -P openshift
+```
+
+### Demo
+
+This profiles self isolate the application from any third party dependencies
```

Comment:
again, change to either:
`This profile self isolates the application from any third party dependencies`
OR
`These profiles self isolate the application from any third party dependencies`